contrib/examples/podpresets/podpreset-preset.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: settings.servicecatalog.k8s.io/v1alpha1
```

Comment:
how about just `podpreset.yaml` as the name of this file?

Reply:
I concur.